apps/qubit/modules/jobs/actions/browseAction.class.php

```diff
@@ -59,6 +59,10 @@ public function execute($request)
             $criteria->add(QubitJob::STATUS_ID, QubitTerm::JOB_STATUS_IN_PROGRESS_ID);
         }
 
+        if ('failed' === $this->filter) {
```

Comment:
We could use an `else if` here instead to skip this check if the filter is already set to active.